ci/cloudbuild/builds/lib/integration.sh

```diff
@@ -30,6 +30,11 @@ source module ci/lib/io.sh
 export PATH="${HOME}/.local/bin:${PATH}"
 python3 -m pip install --quiet --user -r "${PROJECT_ROOT}/google/cloud/storage/emulator/requirements.txt"
 
+echo "================================================================"
```

Comment:
We've avoided these echo lines in the GCB builds by using `io::log_h2` and friends instead. Consider switching to that, or if you prefer to keep it quieter keep the `io::log_yellow` and just drop this echo line.

Reply:
Done.